streams/test-utils/src/main/java/org/apache/kafka/streams/TopologyTestDriver.java

```diff
@@ -358,6 +363,28 @@ public void pipeInput(final ConsumerRecord<byte[], byte[]> consumerRecord) {
         }
     }
 
+    private void validateSourceTopicNameRegexPattern(String consumerTopic) {
+        for (final String sourceTopicName : internalTopologyBuilder.getSourceTopicNames()) {
+            if (!sourceTopicName.equals(consumerTopic) && Pattern.compile(sourceTopicName).matcher(consumerTopic).matches()) {
+                throw new TopologyException("Topology add source of type String for topic: " + sourceTopicName +
+                        " cannot contain regex pattern for consumer topic: " + consumerTopic +
```

Comment:
not sure, what a `consumer topic` is ? Think we should rephrase this.

Reply:
Fixed. Can you please review `message` string and let me know if any modifications needed.